src/redux/reducers/todoLists.js

```diff
@@ -8,6 +8,7 @@ const initialState = {
        0: {
          id: 0,
          name: "First todo",
+         dueDate: "13/09/2020",
```

Comment:
I think the dueDate should be a date, so `new Date()` (which by default sets the date to today)